packages/application-shell/src/components/navbar/navbar.styles.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+const path = require('path');
+
+const compiledStyles = {
+  global: fs.readFileSync(path.join(__dirname, './compiled/navbar.css'), {
+    encoding: 'utf8',
+  }),
+  jsonMap: require('./compiled/navbar.css.json'),
+};
```

Comment:
This is the macro. We're simply returning the loaded CSS and JSON map.

Note that in theory we could have run `postcss` here as well (instead of using a custom script). However, the process is async and it would complicate things, as we want here things to be sync.